test/config.test.ts

```
@@ -0,0 +1,105 @@
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { describe, it } from 'node:test';
+import assert from 'node:assert';
+
+describe('Config auto-unbundling behavior', () => {
+  it('should auto-enable indexing and workers when verification is enabled', async () => {
+    // Set verification enabled, don't set filters or workers
+    process.env.ENABLE_BACKGROUND_DATA_VERIFICATION = 'true';
+    delete process.env.ANS104_UNBUNDLE_FILTER;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)